build.xml

```diff
@@ -655,10 +657,10 @@
           <dependency groupId="io.airlift" artifactId="airline" version="0.8">
             <exclusion groupId="com.google.code.findbugs" artifactId="jsr305" />
            </dependency>
-          <dependency groupId="io.netty" artifactId="netty-bom" version="4.1.118.Final" type="pom" scope="provided"/>
-          <dependency groupId="io.netty" artifactId="netty-all" version="4.1.118.Final" />
-          <dependency groupId="io.netty" artifactId="netty-transport-native-epoll" version="4.1.118.Final"/>
-          <dependency groupId="io.netty" artifactId="netty-transport-native-epoll" version="4.1.118.Final" classifier="linux-x86_64"/>
+	   <dependency groupId="io.netty" artifactId="netty-bom" version="${netty.version}" type="pom" scope="provided"/>
```

Comment:
looks like you changed the spacing on accident here?

Reply:
Apparently I edited this in `vi` originally and it's set to convert spaces to tabs :facepalm: 
I've fixed it now.